portal/src/graphql/adminapi/UserDetailSummary.module.scss

```diff
@@ -0,0 +1,30 @@
+.root {
+  display: flex;
+  align-content: center;
+  min-width: 600px;
+}
+
+.profilePic {
+  width: 75px;
+  height: 75px;
+  margin: auto 0;
+}
+
+.userData {
+  margin-left: 17px;
+  padding-right: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.userInfo {
+  margin-top: 5px;
+  margin-bottom: 20px;
+}
+
+.userInfoRow {
+  font-size: 13px;
+  font-weight: 300;
+  margin-bottom: 5px;
+}
```

Comment:
We can use CSS grid. Using grid can greatly simplify the HTML markup. Technically we do not need to introduce div for layout purpose.

Let's convert the new layout in this PR to use CSS grid.

Reply:
not sure how to use CSS grid to simplify the layout. If we put all user info into a grid, then we still need to group the rows to two group by div? Also we need to give each row a class name and assign one row in the grid by `grid-row`?